python_client/pyrefinebio/sample.py

```diff
@@ -0,0 +1,204 @@
+    """
+
+    valid_filters = [
+        "ordering",
+        "title",
+        "organism",
+        "source_database",
+        "source_archive_url",
+        "has_raw",
+        "platform_name",
+        "technology",
+        "manufacturer",
+        "sex",
+        "age",
+        "specimen_part",
+        "genotype",
+        "disease",
+        "disease_stage",
+        "cell_line",
+        "treatment",
+        "race",
+        "subject",
+        "compound",
+        "time",
+        "is_processed",
+        "is_public",
+        "limit",
+        "offset",
+        "dataset_id",
+        "experiment_accession_code",
+        "accession_codes",
+    ]
+
+    def __init__(
+        self=None,
+        id=None,
+        title=None,
+        accession_code=None,
+        source_database=None,
+        organism=None,
+        platform_accession_code=None,
+        platform_name=None,
+        pretty_platform=None,
+        technology=None,
+        manufacturer=None,
+        protocol_info=None,
+        annotations=None,
+        results=None,
+        source_archive_url=None,
+        has_raw=None,
+        sex=None,
+        age=None,
+        specimen_part=None,
+        genotype=None,
+        disease=None,
+        disease_stage=None,
+        cell_line=None,
+        treatment=None,
+        race=None,
+        subject=None,
+        compound=None,
+        time=None,
+        is_processed=None,
+        created_at=None,
+        last_modified=None,
+        original_files=None,
+        computed_files=None,
+        experiment_accession_codes=None,
+    ):
+        self.id = id
+        self.title = title
+        self.accession_code = accession_code
+        self.source_database = source_database
+        self.organism = Organism(**organism)
+        self.platform_accession_code = platform_accession_code
+        self.platform_name = platform_name
+        self.pretty_platform = pretty_platform
+        self.technology = technology
+        self.manufacturer = manufacturer
+        self.protocol_info = protocol_info
+        self.annotations = [Annotation(**annotation) for annotation in annotations]
+        self.results = [Result(**result) for result in results]
+        self.source_archive_url = source_archive_url
+        self.has_raw = has_raw
+        self.sex = sex
+        self.age = age
+        self.specimen_part = specimen_part
+        self.genotype = genotype
+        self.disease = disease
+        self.disease_stage = disease_stage
+        self.cell_line = cell_line
+        self.treatment = treatment
+        self.race = race
+        self.subject = subject
+        self.compound = compound
+        self.time = time
+        self.is_processed = is_processed
+        self.created_at = created_at
+        self.last_modified = last_modified
+        self.original_files = original_files
+        self.computed_files = computed_files
+        self.experiment_accession_codes = computed_files
```

Comment:
This seems wrong. Also, I think samples will need an `experiments` field as well. This should use `@property` (see https://www.programiz.com/python-programming/property) to first check if the field is populated, and if it's not use `experiment_accession_codes` to populate the field. This way we won't have to populate the field until the user tries to use it.